includes/admin/load.php

```diff
@@ -330,8 +330,24 @@ static function ( $name ) {
 	}
 
 	if ( isset( $_GET['activate'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$message = __( 'Feature activated.', 'performance-lab' );
+
+		$plugin_settings_link = perflab_get_plugin_settings_link( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		if ( $plugin_settings_link ) {
+			/* translators: %s is the settings URL */
+			$message .= ' ' . sprintf( __( 'Review <a href="%s">settings</a>.', 'performance-lab' ), esc_url( $plugin_settings_link ) );
```

Comment:
Minor point: Let's rename the function to specify that it is returning the URL:

```suggestion
		$plugin_settings_url = perflab_get_plugin_settings_url( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
		if ( $plugin_settings_url ) {
			/* translators: %s is the settings URL */
			$message .= ' ' . sprintf( __( 'Review <a href="%s">settings</a>.', 'performance-lab' ), esc_url( $plugin_settings_url ) );
```